2025-03-08 12:33:43,462 INFO    Thread-8 (_run_job):27280 [wandb_init.py:setup_run_log_directory():647] Logging user logs to K:\DA6401\da6401_assignment1\wandb\run-20250308_123343-2a75hlbk\logs\debug.log
2025-03-08 12:33:43,462 INFO    Thread-8 (_run_job):27280 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to K:\DA6401\da6401_assignment1\wandb\run-20250308_123343-2a75hlbk\logs\debug-internal.log
2025-03-08 12:33:43,463 INFO    Thread-8 (_run_job):27280 [wandb_init.py:init():761] calling init triggers
2025-03-08 12:33:43,464 INFO    Thread-8 (_run_job):27280 [wandb_init.py:init():766] wandb.init called with sweep_config: {'activation': 'ReLU', 'batch_size': 32, 'dataset': 'fashion_mnist', 'epochs': 10, 'hidden_size': 128, 'learning_rate': 0.001, 'loss': 'mean_squared_error', 'num_layers': 5, 'optimizer': 'adam', 'weight_decay': 0.0005, 'weight_init': 'random'}
config: {'_items': {'_wandb': {}, 'activation': 'ReLU', 'batch_size': 32, 'dataset': 'fashion_mnist', 'epochs': 10, 'hidden_size': 128, 'learning_rate': 0.001, 'loss': 'mean_squared_error', 'num_layers': 5, 'optimizer': 'adam', 'weight_decay': 0.0005, 'weight_init': 'random'}, '_locked': {'activation': 0, 'batch_size': 0, 'dataset': 0, 'epochs': 0, 'hidden_size': 0, 'learning_rate': 0, 'loss': 0, 'num_layers': 0, 'optimizer': 0, 'weight_decay': 0, 'weight_init': 0}, '_users': {'sweep': 0}, '_users_inv': {0: 'sweep'}, '_users_cnt': 1, '_callback': <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x00000214B4FF7D90>>, '_settings': Settings(allow_offline_artifacts=True, allow_val_change=False, anonymous=None, api_key=None, azure_account_url_to_access_key=None, base_url='https://api.wandb.ai', code_dir=None, config_paths=None, console='wrap', console_multipart=False, credentials_file='C:\\Users\\ANJALI VERMA\\.config\\wandb\\credentials.json', disable_code=False, disable_git=False, disable_job_creation=True, docker=None, email='da24m002@smail.iitm.ac.in', entity='da24m002-indian-institute-of-technology-madras', organization=None, force=False, fork_from=None, git_commit=None, git_remote='origin', git_remote_url=None, git_root=None, heartbeat_seconds=30, host='LAPTOP-QS3R10DF', http_proxy=None, https_proxy=None, identity_token_file=None, ignore_globs=(), init_timeout=90.0, insecure_disable_ssl=False, job_name=None, job_source=None, label_disable=False, launch=False, launch_config_path=None, login_timeout=None, mode='online', notebook_name=None, program='K:\\DA6401\\da6401_assignment1\\train.py', program_abspath='K:\\DA6401\\da6401_assignment1\\train.py', program_relpath='train.py', project='myprojectname', quiet=False, reinit=False, relogin=False, resume=None, resume_from=None, resumed=False, root_dir='K:\\DA6401\\da6401_assignment1', run_group=None, run_id='2a75hlbk', run_job_type=None, run_name='honest-sweep-3', run_notes=None, run_tags=None, sagemaker_disable=False, save_code=None, settings_system='C:\\Users\\ANJALI VERMA\\.config\\wandb\\settings', show_colors=None, show_emoji=None, show_errors=True, show_info=True, show_warnings=True, silent=False, start_method=None, strict=None, summary_timeout=60, summary_warnings=5, sweep_id='lmepeowr', sweep_param_path='K:\\DA6401\\da6401_assignment1\\wandb\\sweep-lmepeowr\\config-2a75hlbk.yaml', symlink=False, sync_tensorboard=None, table_raise_on_max_row_limit_exceeded=False, username=None, x_cli_only_mode=False, x_disable_meta=False, x_disable_service=False, x_disable_setproctitle=False, x_disable_stats=False, x_disable_viewer=False, x_disable_machine_info=False, x_executable='C:\\Users\\ANJALI VERMA\\anaconda3\\envs\\dl\\python.exe', x_extra_http_headers=None, x_file_stream_max_bytes=None, x_file_stream_max_line_bytes=None, x_file_stream_transmit_interval=None, x_file_stream_retry_max=None, x_file_stream_retry_wait_min_seconds=None, x_file_stream_retry_wait_max_seconds=None, x_file_stream_timeout_seconds=None, x_file_transfer_retry_max=None, x_file_transfer_retry_wait_min_seconds=None, x_file_transfer_retry_wait_max_seconds=None, x_file_transfer_timeout_seconds=None, x_files_dir=None, x_flow_control_custom=None, x_flow_control_disabled=None, x_graphql_retry_max=None, x_graphql_retry_wait_min_seconds=None, x_graphql_retry_wait_max_seconds=None, x_graphql_timeout_seconds=None, x_internal_check_process=8.0, x_jupyter_name=None, x_jupyter_path=None, x_jupyter_root=None, x_label=None, x_live_policy_rate_limit=None, x_live_policy_wait_time=None, x_log_level=20, x_network_buffer=None, x_primary=True, x_proxies=None, x_runqueue_item_id=None, x_require_legacy_service=False, x_save_requirements=True, x_service_transport=None, x_service_wait=30.0, x_start_time=1741417422.7253573, x_stats_pid=27280, x_stats_sampling_interval=10.0, x_stats_neuron_monitor_config_path=None, x_stats_dcgm_exporter=None, x_stats_open_metrics_endpoints=None, x_stats_open_metrics_filters=None, x_stats_open_metrics_http_headers=None, x_stats_disk_paths=('/',), x_stats_gpu_device_ids=None, x_stats_buffer_size=0, x_sync=False, x_update_finish_state=True, colab_url=None, deployment='cloud', files_dir='K:\\DA6401\\da6401_assignment1\\wandb\\run-20250308_123342-2a75hlbk\\files', is_local=False, log_dir='K:\\DA6401\\da6401_assignment1\\wandb\\run-20250308_123342-2a75hlbk\\logs', log_internal='K:\\DA6401\\da6401_assignment1\\wandb\\run-20250308_123342-2a75hlbk\\logs\\debug-internal.log', log_symlink_internal='K:\\DA6401\\da6401_assignment1\\wandb\\debug-internal.log', log_symlink_user='K:\\DA6401\\da6401_assignment1\\wandb\\debug.log', log_user='K:\\DA6401\\da6401_assignment1\\wandb\\run-20250308_123342-2a75hlbk\\logs\\debug.log', project_url='https://wandb.ai/da24m002-indian-institute-of-technology-madras/myprojectname', resume_fname='K:\\DA6401\\da6401_assignment1\\wandb\\wandb-resume.json', run_mode='run', run_url='https://wandb.ai/da24m002-indian-institute-of-technology-madras/myprojectname/runs/2a75hlbk', settings_workspace='K:\\DA6401\\da6401_assignment1\\wandb\\settings', sweep_url='https://wandb.ai/da24m002-indian-institute-of-technology-madras/myprojectname/sweeps/lmepeowr', sync_dir='K:\\DA6401\\da6401_assignment1\\wandb\\run-20250308_123342-2a75hlbk', sync_file='K:\\DA6401\\da6401_assignment1\\wandb\\run-20250308_123342-2a75hlbk\\run-2a75hlbk.wandb', sync_symlink_latest='K:\\DA6401\\da6401_assignment1\\wandb\\latest-run', timespec='20250308_123342', wandb_dir='K:\\DA6401\\da6401_assignment1\\wandb\\'), '_artifact_callback': <bound method Run._config_artifact_callback of <wandb.sdk.wandb_run.Run object at 0x00000214B4FF7D90>>, '_wandb': {}}
2025-03-08 12:33:43,465 INFO    Thread-8 (_run_job):27280 [wandb_init.py:init():776] wandb.init() called while a run is active
2025-03-08 12:33:44,025 INFO    Thread-8 (_run_job):27280 [wandb_run.py:_finish():2112] finishing run da24m002-indian-institute-of-technology-madras/myprojectname/2a75hlbk
2025-03-08 12:33:44,026 INFO    Thread-8 (_run_job):27280 [wandb_run.py:_atexit_cleanup():2342] got exitcode: 1
2025-03-08 12:33:44,027 INFO    Thread-8 (_run_job):27280 [wandb_run.py:_restore():2324] restore
2025-03-08 12:33:44,028 INFO    Thread-8 (_run_job):27280 [wandb_run.py:_restore():2330] restore done
2025-03-08 12:33:46,340 INFO    MsgRouterThr:27280 [mailbox.py:close():115] Closing mailbox, abandoning 1 handles.
2025-03-08 12:33:46,340 INFO    Thread-8 (_run_job):27280 [wandb_run.py:_footer_history_summary_info():3958] rendering history
2025-03-08 12:33:46,341 INFO    Thread-8 (_run_job):27280 [wandb_run.py:_footer_history_summary_info():3990] rendering summary
2025-03-08 12:33:46,341 INFO    Thread-8 (_run_job):27280 [wandb_run.py:_footer_sync_info():3919] logging synced files
2025-03-08 12:33:46,355 ERROR   MainThread:27280 [pyagent.py:_run_jobs_from_queue():234] Run 2a75hlbk errored:
Traceback (most recent call last):
  File "C:\Users\ANJALI VERMA\anaconda3\envs\dl\lib\site-packages\wandb\sdk\wandb_config.py", line 165, in __getattr__
    return self.__getitem__(key)
  File "C:\Users\ANJALI VERMA\anaconda3\envs\dl\lib\site-packages\wandb\sdk\wandb_config.py", line 130, in __getitem__
    return self._items[key]
KeyError: 'beta1'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\ANJALI VERMA\anaconda3\envs\dl\lib\site-packages\wandb\agents\pyagent.py", line 306, in _run_job
    self._function()
  File "K:\DA6401\da6401_assignment1\train.py", line 163, in sweep_train
    train_model(wandb.config)
  File "K:\DA6401\da6401_assignment1\train.py", line 101, in train_model
    optimizer_params["beta1"] = config.beta1
  File "C:\Users\ANJALI VERMA\anaconda3\envs\dl\lib\site-packages\wandb\sdk\wandb_config.py", line 167, in __getattr__
    raise AttributeError(
AttributeError: <class 'wandb.sdk.wandb_config.Config'> object has no attribute 'beta1'

2025-03-08 12:33:46,394 ERROR   MainThread:27280 [pyagent.py:_run_jobs_from_queue():245] Detected 3 failed runs in the first 60 seconds, killing sweep.
